### ISSUE TITLE
webpacker - Unknown argument: host

### ISSUE THREAD

**Issue by webdevsimon** [madwire/docker-compose-rails-template#3]
The command in the docker compose uses `webpack-dev-server` which isn't in the `bin` directory

docker-compose.yml
```
webpacker:
...
command: ./bin/webpack-dev-server --host localhost
# changed to
command: webpack-dev-server --host localhost
```
This changed has resolved my previous problem.

**Comment by bcardiff** [comment]
While trying to get rails 5.1.4 with webpack hmr I arrived here. I use `bundle exec ./bin/webpack-dev-server` because otherwise I was getting a missing rake related error. But the ./bin/webpack-dev-server binstub does not support command line arguments.

If changing the host in the webpacker.yml is not desired[1], another option to pass `--host` is to use the `WEBPACKER_DEV_SERVER_HOST=` environment variable.

---

[1] someone might not want to change the webpacker.yml host to be friendly in non dockerized development environment, and use the environment variables to override things if using docker.

**Comment by madwire** [comment]
@webdevsimon @bcardiff Thanks both, so what is the combined solution to this issue.

**Comment by madwire** [comment]
Are we saying that this is the right command:
```yaml
command: bundle exec ./bin/webpack-dev-server --host WEBPACKER_DEV_SERVER_HOST
```
And  a tweak to the dockerfile to contain a default WEBPACKER_DEV_SERVER_HOST

**Comment by bcardiff** [comment]
@madwire, no. `./bin/webpack-dev-server` does not accept arguments.
The webpacker read the host from the webpacker.yml, or if provided, from the `WEBPACKER_DEV_SERVER_HOST` env var. The second with higher preference.

I use that and others variable when launching docker to override the localhosts of db and now webpacker.

**Comment by bcardiff** [comment]
@madwire It's a fresh project, so things make change, but here is the boilerplate I am using to work with docker. https://github.com/instedd/aida-ui

**Comment by madwire** [comment]
fixed with #4
